docs/cli/cli.md

```diff
@@ -0,0 +1,122 @@
+# 💻 CLI for `jstz`
+
+This guide will instruct through how to use the command line interface for `jstz`.
+
+## Overview
+
+`jstz` provides a simple command line interface where the user can easily run the sandbox environment and test smart functions.
+
+## Basic Commands
+
+- `jstz <COMMAND>`: The primary way to interact with `jstz`.
+- `help`: Displays a general help message or help information for specific subcommands.
+- `--version (-V)`: Displays the current version of `jstz`.
+
+## Sandbox
+
+The sandbox commands are responsible for managing the `jstz` sandbox environment.
+
+### Usage:
+
+```bash
+jstz sandbox <COMMAND>
+```
+
```

Comment:
The usage block here is redundant, it would be best to have sections for each command with a full usage for each command, explaining the arguments (and their types) and maybe some example output from running the command